packages/contracts/src/Game.sol

```diff
@@ -161,6 +161,7 @@ contract Game {
         uint16 health;
         uint8 deckStart;
         uint8 deckEnd;
+        uint8 handSize;
```

Comment:
So the idea was that we could keep that private, but we'd have to check if `handSize == maxHandSize || maxSizedHand[handSize] == NO_CARD` in the proof, right?

Definitely not worth doing now, since cards can be counted anyway. In the future, not knowing the numbers of cards is kinda funny, but not super duper needed either.